src/test/java/duke/command/CommandTest.java

```diff
@@ -0,0 +1,38 @@
+package duke.command;
+
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.Optional;
+import java.util.regex.Matcher;
+
+import org.junit.jupiter.api.Test;
+
+public class CommandTest {
+    private void testCommandRejectsBadInputs(Command command) {

```

Comment:
Perhaps you can consider using the following naming format "whatIsBeingTested_descriptionOfTestInputs_expectedOutcome" for your test cases?

Reply:
Good suggestion! Forgot about this style of naming tests.